commit 9af31d021e83dce4e98f06d8e732f4afb481cd08
Author: Daniel Martinez <danielmartinez@cock.li>
Date:   Sat Aug 3 18:04:04 2024 -0400

    disable lldb docs as they depend on some sphinx packages we don't have

diff --git a/lldb/CMakeLists.txt b/lldb/CMakeLists.txt
index 59cdc4593463..e7c2c38b50a9 100644
--- a/lldb/CMakeLists.txt
+++ b/lldb/CMakeLists.txt
@@ -124,7 +124,6 @@ add_subdirectory(utils/TableGen)
 
 add_subdirectory(source)
 add_subdirectory(tools)
-add_subdirectory(docs)
 
 if (LLDB_ENABLE_PYTHON)
   if(LLDB_BUILD_FRAMEWORK)
